tests/SpecTests/AtlasDataLakeSpecTest.php

```diff
@@ -237,13 +240,8 @@ public function testConnectwithSCRAMSHA256(): void
 
     private function isAtlasDataLake(): bool
     {
-        $cursor = $this->manager->executeCommand(
-            $this->getDatabaseName(),
-            new Command(['buildInfo' => 1])
-        );
-
-        $document = current($cursor->toArray());
+        $isDataLake = getenv('MONGODB_IS_DATA_LAKE');
```

Comment:
Please add a new section to the "Environment Variables" section in CONTRIBUTING.md:

```
The following environment variable is used for [Atlas Data Lake testing](https://github.com/mongodb/specifications/blob/master/source/atlas-data-lake-testing/tests/README.rst):

 * `MONGODB_IS_DATA_LAKE`: Specify a true boolean string
   (see: [`FILTER_VALIDATE_BOOLEAN`](https://www.php.net/manual/en/filter.filters.validate.php))
   if `MONGODB_URI` points to an Atlas Data Lake instance. Defaults to false.
```

I don't think we need additional prose (like we have for serverless) in the paragraphs above directly under "Testing".

Reply:
Added, thank you for providing the wording!